SetupScript.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Permission Denied"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c3 google.com > /dev/null
+
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+cd /opt/
```

Comment:
Using `cd` in these scripts is very confusing. Call subshells with `cd` instead.

```bash
(cd /opt/ && git clone https://github.com/UCSolarCarTeam/EspilonRaspberry.git)
```

It isn't less code... but then the reader doesn't need to keep track of where they are in the filesystem :)